<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugControl</class>
 <widget class="QDialog" name="DebugControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugControl</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>excution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startstop">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cont">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="step">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stepIn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>stepIn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stepOut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>stepOut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>breakpoint:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBp">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBp">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>info:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>error:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="errorLog">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="input"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>breakpoints:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="breakpoints"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>pending breakpoints:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="pendingBp"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
